test/test-functions

```diff
@@ -187,6 +187,7 @@ BASICTOOLS=(
     login
     losetup
     lz4cat
+    mkdir
```

Comment:
I do not know why this is not installed. Is this necessary? cc @mrc0mmand

Reply:
Ditto (part of `$DEBUGTOOLS` below).

Reply:
It doesn't make much sense (`mkdir` is a basic tool, imo), but that's something I'll look into later.